test/Limiters/limiter.jl

```diff
@@ -113,23 +128,25 @@ end
         limiter = Limiters.QuasiMonotoneLimiter(ρq)
         Limiters.compute_bounds!(limiter, ρq, ρ)
 
+        is_cpu = device isa ClimaComms.AbstractCPUDevice
         for h2 in 1:n2
             for h1 in 1:n1
                 s = slab(limiter.q_bounds, h1 + n1 * (h2 - 1))
-                q_min = s[1]
-                q_max = s[2]
-                @test q_min.x ≈ 2 * (h1 - 1)
-                @test q_min.y ≈ 3 * (h2 - 1)
-                @test q_max.x ≈ 2 * h1
-                @test q_max.y ≈ 3 * h2
+                # @show s
+                CUDA.@allowscalar q_min = s[1]
+                CUDA.@allowscalar q_max = s[2]
+                is_cpu && @test q_min.x ≈ 2 * (h1 - 1)
+                is_cpu && @test q_min.y ≈ 3 * (h2 - 1)
+                is_cpu && @test q_max.x ≈ 2 * h1
+                is_cpu && @test q_max.y ≈ 3 * h2
```

Comment:
It seems to me that all these tests are relevant only for the cpu case. It's that's the case, we could move the if statement before the two for loops. (But, since you haven't done it, I am guessing I am missing something...).

Reply:
Yeah, my plan is to remove these conditionals once I add the implementation in

Reply:
Sounds good!